data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/request/command/RevokePreAuthToken.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "RevokePreAuthToken",
+  "namespace": "net.corda.data.membership.db.request.command",
+  "doc": "Revoke a pre auth token",
+  "fields": [
+    {
+      "name": "tokenId",
+      "doc": "Token identifier of the pre auth token.",
+      "type": "string"
+    },
+    {
+      "name": "remark",
+      "doc": "Optimal remark about why the token was revoked.",
```

Comment:
```suggestion
      "doc": "Optional remark about why the token was revoked.",
```

Reply:
I haven't reviewed the OS behaviour yet but I'm curious how is best to handle remarks since we can set them during add and during revoke. Will it be the case that the revoke remark overwrites the create remark (potentially losing context of why the remark was created). This may be answered elsewhere but leaving this comment so I don't forget to check that case.
This same question applies for auto invalidated tokens, or consumed token if we set a remark then also.

Reply:
I think we should add another column for the a remark when the token gets consumed, revoked or auto invalidated. That would better than appending.